src/app.js

```diff
@@ -17,13 +18,18 @@ const io = new Server(httpServer, {
 
 const dynamo = new Dynamo()
 
-setInterval(async () => await dynamo.deleteStaleUsers(), heartbeatInterval)
+setInterval(async () => await dynamo.deleteStaleUsers(), HEARTBEAT_INTERVAL)
 
 io.of('/game').on('connection', (socket) => {
   console.log(`Connected with client ${socket.id}`)
   socket.on(JOIN_ROOM, (roomId) => {
     socket.join(roomId)
     console.log(`Client ${socket.id} is joined to room ${roomId}!`)
+    io.of('/game').to(roomId).emit(GAME_STATE, mockGameStateEventArgs)
+  })
+  socket.on(MOVE_PLAYED, (movePlayedEventArgs) => {
+    //TODO: a method for updating game_state data
+    io.of('/game').to(mockGameStateEventArgs.id).emit(GAME_STATE, mockGameStateEventArgs)
```

Comment:
instead of sending `to()` `mockGameStateEventArgs.id`, we can now send `movePlayedEventArgs.gameId`